### PR TITLE
[s2i] use source to image

### PR DESCRIPTION
**Pull request by mikz**
- [x] use s2i
- [ ] download s2i on travis

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,12 +19,12 @@ dependencies:
 	luarocks make --local rockspec
 
 build:
-	$(DOCKER_COMPOSE) build
+	$(S2I) build . openresty-builder docker-gateway-test
```
Comment (jmprusi) [inline Makefile]: should we download the openresty-builder and build it locally or depend on the quay.io/3scale/openresty-builder:SOME_VERSION ? 

I would go for the second option.

Comment (mikz) [inline Makefile]: I'd much rather depend on some published version. At least for now it would make things much easier.

Comment (jmprusi) [inline Makefile]: Ok, we can publish the openresty-builder manually.. then look for a way to automate it.. just to unblock this.

Comment (mikz) [inline Makefile]: I'll automate it if I can find 30 minutes between meetings and a pub later.

**Comment by jmprusi** [comment]
👍

**Pull request merged**